ui/guest/src/store/epics/root.ts

```diff
@@ -124,7 +123,7 @@ const epic = combineEpics<GuestStandardAction, GuestStandardAction, GuestState>(
       ofType('mouseover', 'mouseleave'),
       withLatestFrom(state$),
       filter((args) => args[1].status === EditingStatus.LISTENING),
-      tap(([action]) => action.payload.event.stopPropagation()),
+      tap(([action]) => action.payload.event.stopPropagation?.()),
```

Comment:
Why would an event not have the stopProp?

Reply:
It probably was a wrong type at some moment, removing the optional chaining